.circleci/config.yml

```diff
@@ -64,7 +64,7 @@ jobs:
           name: Build docs to store
           command: |
             cd docs
-            make html
+            make metrics html
```

Comment:
I'm a bit confused about this, it seems like the intention is to run two separate make commands `make metrics` and `make html`, but isn't the `make metrics html` going to lead to a `make metrics` command being invoked with a passed argument of `html`?

Reply:
I experimented and concluded that `make metrics rest-api` would for example first run make metrics and then make rest-api. All good with this logic.